broker/src/models/call_log.erl

```diff
@@ -12,3 +12,8 @@ info(Message, Details, #call_log{id = CallId}) ->
 trace(Message, Details, #call_log{id = CallId}) ->
   call_log_entry:create("trace", CallId, Message, Details).
 
+cancel_active_calls_for_minutes(N, Reason) ->
+  N_Minutes_Ago = util:seconds_ago(N * 60),
+  Count = call_log:update_all([{state, "failed"}, {fail_reason, Reason}], [{state, "active"}, {started_at, '<', N_Minutes_Ago}]),
```

Comment:
The `fail_reason` field seems to be kind of pre-fixed (they look like symbols, like `no-answer`, `no-call-flow` or `internal error`). Think of it as a field by which you can easily index/search for calls.

And then you can be a bit more verbose on the `fail_details`. I'd set both of them here 👍